internal/graph/singleflight_resolver.go

```diff
@@ -78,6 +84,9 @@ func (s *singleflightCheckResolver) ResolveCheck(
 		resp.ResolutionMetadata.DatastoreQueryCount = 0
 	}
 
+	span.SetAttributes(attribute.Bool("is_shared_request", shared))
+	span.SetAttributes(attribute.Bool("is_ultimately_processed_request", isUnique))
```

Comment:
What's the importance of this for our usage in our telemetry? Can we approach this with the suggestion in https://github.com/openfga/openfga/pull/1331/files#r1474847198 instead?